### ISSUE TITLE
Add support for categorical response in blm

### ISSUE THREAD

**Issue by bgroenks96** [cpfiffer/BayesModels#6]
Requires a separate `lm` implementation (I think) due to the dummy coded response variable being multivariate.

**Comment by just-cameron** [comment]
Agreed, though I'm not sure what exactly the solution is here. I'm not super familiar with categorical response regressions.

**Comment by bgroenks96** [comment]
Oh, in case it wasn't clear, I just mean plain ol' [multinomial logistic regression](https://en.wikipedia.org/wiki/Multinomial_logistic_regression). Maybe I got my terms mixed up there.

Interestingly, the [Turing tutorial](https://turing.ml/dev/tutorials/8-multinomiallogisticregression/) uses an ordinal encoding, which is still univariate. Does StatsModels support this? It seems to generate a one-hot encoded response variable by default.

**Comment by just-cameron** [comment]
It does for the RHS, not for the LHS as far as I can tell. All we'd need is something that would convert something to integers indexing categories for it to work with the softmax/Categorical set up in the tutorial.

@kleinschmidt do you know how to get StatsModels to convert response variables into categorical codings?

**Comment by kleinschmidt** [comment]
Hm, that's a good question. I think you could implement a contrasts scheme for ordinal encoding (or even use `ContrastsCoding` with `[1:length(levels(x)); ]` as the contrasts as a quick and dirty workaround.

**Comment by kleinschmidt** [comment]
Or you should be able to just pass in a vector of `Int` for that column and it will be treated as continuous, but that's not so satisfying.

**Comment by rookiecdn** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.
steal-7126, @fly2sky2018, @oclemens2000, @klin10, @marley1989, @rimallo, @eugenelab, @dukehao, @darxtrix, @dbSOD, @nxtreaming, @siyuano, @nebotfj, @pjedwabny, @jaxontn, @kinok, @Keekuk, @nspassov, @rafflesgun, @deantzzz
